.changeset/funny-mirrors-explode.md

```diff
@@ -0,0 +1,9 @@
+---
+'@finos/legend-art': patch
+'@finos/legend-extension-dsl-persistence': patch
+'@finos/legend-graph-extension-collection': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio-app': patch
+---
```

Comment:
````suggestion
'@finos/legend-extension-dsl-persistence': major
'@finos/legend-graph-extension-collection': minor
'@finos/legend-manual-tests': minor
'@finos/legend-studio-app': minor
````

Reply:
You can move the legend-art to a separate change set file with no message.

Reply:
Done and committed your suggestion.